### ISSUE TITLE
[MAS2.1.1][Keyboard Navigation- Welcome Tab Screen] "X" button of already created Bot in "My Bot" section is not accessible with keyboard.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1952]
**Actual Result:**
"X" button of already created Bot in "My Bot" section is not accessible with keyboard whereas same can be access by mouse.
The "X" button didn't gets the keyboard focus.

**Expected Result:**
Every control which can be accessible via mouse should be accessible via keyboard.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "My Bot" section and press Tab to reach any created Bot.
4. Observe the issue.

**User Impact:**
When a component can't be accessible by keyboard the keyboard dependent user will not able to activate the component using the keyboard as well as user will not be able to use the functionality of the control. 

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D](url)

**Attachment:**
[X button is not accessible with keyboard.zip](https://github.com/microsoft/BotFramework-Emulator/files/3794146/X.button.is.not.accessible.with.keyboard.zip)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-94878
